WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -479,10 +514,44 @@ public void onStart() {
     }
 
     @SuppressWarnings("unused")
-    public void onEventMainThread(NotificationEvents.NoteModerationStatusChanged event) {
-        setNoteIsModerating(event.mNoteId, event.mIsModerating);
+    public void onEventMainThread(NotificationEvents.NotificationsRefreshError error) {
+        if (isAdded()) {
+            ToastUtils.showToast(getActivity(), getString(R.string.error_refresh_notifications));
+            mSwipeRefreshLayout.setRefreshing(false);
+        }
+    }
+    @SuppressWarnings("unused")
+    public void onEventMainThread(final NotificationEvents.NotificationsRefreshCompleted event) {
+        if (!isAdded()) {
+            return;
+        }
+        mSwipeRefreshLayout.setRefreshing(false);
+        mNotesAdapter.addAll(event.notes, true);
+    }
 
-        EventBus.getDefault().removeStickyEvent(NotificationEvents.NoteModerationStatusChanged.class);
+    @SuppressWarnings("unused")
+    public void onEventMainThread(final NotificationEvents.NoteModerationStatusChanged event) {
+        if (event.mIsModerating) {
+            setNoteIsModerating(event.mNoteId, event.mIsModerating);
```

Comment:
Is it normal to have a an `m` named member used in this way? I always considered them to be private, so you would use something like `event.noteId` here.